hi-ml-azure/src/health_azure/himl.py

```diff
@@ -607,10 +607,23 @@ def _generate_azure_datasets(
     :param cleaned_output_datasets: The list of output dataset configs
     :return: The AzureRunInfo containing the AzureML input and output dataset lists etc.
     """
-    returned_input_datasets = [Path(RUN_CONTEXT.input_datasets[_input_dataset_key(index)])
-                               for index in range(len(cleaned_input_datasets))]
-    returned_output_datasets = [Path(RUN_CONTEXT.output_datasets[_output_dataset_key(index)])
-                                for index in range(len(cleaned_output_datasets))]
+    if is_amulet_job():
+        input_data_mount_folder = Path(os.environ[ENV_AMLT_DATAREFERENCE_DATA])
+        print(f"Path to mounted data: {ENV_AMLT_DATAREFERENCE_DATA}: {str(input_data_mount_folder)}")
```

Comment:
Intentional to print out the name of the environment variable?

Reply:
(also switch from print to logging.info for consistency?)

Reply:
Yes intentional - updated